extensions/schema-compare/package.json

```diff
@@ -1,52 +1,57 @@
 {
-  "name": "schema-compare",
```

Comment:
seems unintentional :)

Reply:
Doesn't seem like there were any changes
This change was probably because of the format document command.

Reply:
There's this _metadata tag added. Dunno what that's about but you should revert this regardless since it's not related to your changes

---
In reply to: [285718197](https://github.com/microsoft/azuredatastudio/pull/5542#discussion_r285718197) [](ancestors = 285718197)